channels.go

```diff
@@ -0,0 +1,134 @@
+package telebot
+
+import (
+	"encoding/json"
+	"fmt"
+)
+
+// ChatInfo struct is the main chat information from telegram
+type ChatInfo struct {
+	ID       int64  `json:"id"`
+	Title    string `json:"title"`
+	Username string `json:"username"`
+	Type     string `json:"type"`
+}
+
+// For each member of a chat , we have some information like usertype (admin , etc..) 
+// and the main information about user like id and etc...
+type ChatMember struct {
+	User   User   `json:"user"`
+	Status string `json:"status"`
+}
+
+// GetChat allows to get main information of chat like channel username and etc...
+func (b *Bot) GetChat(recipient Recipient) (ChatInfo, error) {
```

Comment:
Same issue here as with getFile. The [getChat](https://core.telegram.org/bots/api#getchat) API call takes a `chat_id` and returns a `Chat` object from this, but this Go implementation works completely different. I'd like to see it behave like the Telegram API does.